cdap-docs/reference-manual/source/http-restful-api/lifecycle.rst

```diff
@@ -604,42 +631,146 @@ For Workflows, you can also retrieve:
-   * - 
-     - 
-   * - HTTP Method
-     - ``GET <base-url>/namespaces/default/apps/PurchaseHistory/workflows/PurchaseHistoryWorkflow/nextruntime``
-   * - Description
-     - Retrieves the next runtime of the Workflow *PurchaseHistoryWorkflow* of the Application *PurchaseHistory*
-   * - Returns
-     - ``[{"id":"DEFAULT.WORKFLOW:developer:PurchaseHistory:PurchaseHistoryWorkflow:0:DailySchedule","time":1415102400000}]``
 
+.. container:: table-block-example
+
+  .. list-table::
+     :widths: 99 1
+     :stub-columns: 1
+
+     * - Example: Retrieving A Schedule
+       - 
+       
+  .. list-table::
+     :widths: 15 85
+     :class: triple-table
+
+     * - Description
+       - Retrieves the schedules of the Workflow *PurchaseHistoryWorkflow* of the Application *PurchaseHistory*
+      
+     * - HTTP Method
+       - ``GET <base-url>/namespaces/default/apps/PurchaseHistory/workflows/PurchaseHistoryWorkflow/schedules``
+         
+     * - Returns
+       - | ``[{"schedule":{"name":"DailySchedule","description":"DailySchedule with crontab 0 4 * * *","cronEntry":"0 4 * * *"},``
+         | `` "program":{"programName":"PurchaseHistoryWorkflow","programType":"WORKFLOW"},"properties":{}}]``
+         
+
+.. container:: table-block-example
+
+  .. list-table::
+     :widths: 99 1
+     :stub-columns: 1
+
+     * - Example: Retrieving The Next Runtime
+       - 
+       
+  .. list-table::
+     :widths: 15 85
+     :class: triple-table
+
+     * - Description
+       - Retrieves the next runtime of the Workflow *PurchaseHistoryWorkflow* of the Application *PurchaseHistory*
+      
+     * - HTTP Method
+       - ``GET <base-url>/namespaces/default/apps/PurchaseHistory/workflows/PurchaseHistoryWorkflow/nextruntime``
+         
+     * - Returns
+       - | ``[{"id":"DEFAULT.WORKFLOW:developer:PurchaseHistory:PurchaseHistoryWorkflow:0:DailySchedule","time":1415102400000}]``
+       
+
+Schedules and Workflows: Suspend and Resume
+...........................................
+
+For both Schedules and Workflows, you can suspend and resume them using the RESTful API.
+
+To *suspend* means that the current activity will be taken to completion, but no further 
+programs will be initiated. Programs will not be left partially uncompleted, barring any errors.
+
+In the case of a Workflow with multiple MapReduce programs, if one of them is running (first of
+three perhaps) and you suspend the Workflow, that first MapReduce will be completed but the
+following two will not be started.
+
+To *resume* means that activity will start up where it was left off, beginning with the start
+of the next program in the sequence.
+
+In the case of the Workflow mentioned above, resuming it after suspension would start up with the
+second of the three MapReduce programs, which is where it would have left off when it was suspended.
+
+With Workflows, *suspend* and *resume* require a *run-id* as the action takes place on
+either a currently running or suspended Workflow.
+
```

Comment:
The above description from `To *suspend* means that the current activity ...` is only applicable to the Workflows. For Schedules, suspending it would mean the program associated with that schedule will not trigger again until the schedule is resumed.